config.m4

```diff
@@ -3,46 +3,53 @@ PHP_ARG_WITH(v8js, for V8 Javascript Engine,
 
 if test "$PHP_V8JS" != "no"; then
   SEARCH_PATH="/usr/local /usr"
-  SEARCH_FOR="libv8.$SHLIB_SUFFIX_NAME"
-
-  if test -d "$PHP_V8JS"; then
-    SEARCH_PATH="$PHP_V8JS"
-
-    # set rpath, so library loader picks up libv8 even if it's not on the
-    # system's library search path
-    case $host_os in
-      darwin* )
-        # MacOS does not support --rpath
-        ;;
-      * )
-        LDFLAGS="$LDFLAGS -Wl,--rpath=$PHP_V8JS/$PHP_LIBDIR"
-        ;;
-    esac
-  fi
-
-  AC_MSG_CHECKING([for V8 files in default path])
-  ARCH=$(uname -m)
-
-  for i in $SEARCH_PATH ; do
-    if test -r "$i/$PHP_LIBDIR/$SEARCH_FOR" -a -r "$i/include/v8/v8.h"; then
-      V8_INCLUDE_DIR="$i/include/v8"
-      V8_LIBRARY_DIR="$i/$PHP_LIBDIR"
-      AC_MSG_RESULT(found in $i)
-    fi
 
-    # Debian installations
-    if test -r "$i/$PHP_LIBDIR/$ARCH-linux-gnu/$SEARCH_FOR"; then
-      V8_INCLUDE_DIR="$i/include/v8"
-      V8_LIBRARY_DIR="$i/$PHP_LIBDIR/$ARCH-linux-gnu"
-      AC_MSG_RESULT(found in $i)
+  for libname in v8 node; do
+    SEARCH_FOR="lib$libname.$SHLIB_SUFFIX_NAME"
+
+    if test -d "$PHP_V8JS"; then
+      SEARCH_PATH="$PHP_V8JS"
+
+      # set rpath, so library loader picks up libv8 even if it's not on the
+      # system's library search path
+      case $host_os in
+        darwin* )
+          # MacOS does not support --rpath
+          ;;
+        * )
+          LDFLAGS="$LDFLAGS -Wl,--rpath=$PHP_V8JS/$PHP_LIBDIR"
+          ;;
+      esac
     fi
 
-    # Manual installations
-    if test -r "$i/$PHP_LIBDIR/$SEARCH_FOR" -a -r "$i/include/libplatform/libplatform.h"; then
-      V8_INCLUDE_DIR="$i/include"
-      V8_LIBRARY_DIR="$i/$PHP_LIBDIR"
-      AC_MSG_RESULT(found in $i)
-    fi
+    AC_MSG_CHECKING([for V8 files in default path])
+    ARCH=$(uname -m)
+
+    for i in $SEARCH_PATH ; do
+    echo checking "$i/$PHP_LIBDIR/$SEARCH_FOR" -a -r "$i/include/$libname/v8.h"
```

Comment:
There are no other echos here. Is this a remaining debug snippet or is it seen helpful for the overall output of the script?

Reply:
yes, that shouldn't actually go there...